elastic/kibana
# [Logs+] End onboarding wizard in default discover

closes [#163080](https://github.com/elastic/kibana/issues/163080)


## 📝  Summary

This PR navigates to default discover at the end of the onboarding flow for both custom and system workflows.
It also adds `logs-*` as the default dataview along with a preset filter for the intended dataset during the onboarding flow.

## ✅  Testing

1. Navigate to the onboarding flow `/app/observabilityOnboarding/`
2. Choose either System logs or Stream log files
3. Go through the onboarding wizard
4. Click the Explore logs button at the end
5. Observe the DataView and Preset Filter after being navigated to Discover

## 🎥 Demo

https://github.com/elastic/kibana/assets/11225826/5eff74e4-c12a-46e7-968a-6efa34a6a7a9
